docker/irods_client/tests/test_search.py

```diff
@@ -58,3 +58,25 @@ def test_find_meta(session, item_name, request, metadata, is_found):
 
     res = _found(search_data(session, metadata=metadata), ipath)
     assert res == is_found
+
+@mark.parametrize("metadata,is_found", [
```

Comment:
Why is this a separate test from the above? Also, I would like to see some tests for the opposite, where we not expect the path to be found because of case sensitiveness.